.vscode/settings.json

```diff
@@ -1,5 +1,5 @@
 {
-  "typescript.validate.enable": false,
-  "javascript.validate.enable": false,
+  "typescript.validate.enable": true,
+  "javascript.validate.enable": true,
```

Comment:
So, keep it off for JS files and use the linting/flow tools from the CLI until we're through this? 👍

Reply:
The Flow extension recommends turning off the JS validation because it seems to use the TS compiler which then complains about Flow type syntax being invalid.  I'm happy leaving the JS validation off, but the TS validation seems to work Ok.